pkg/game/test/factories.go

```diff
@@ -21,3 +21,25 @@ func GetTestScoreReport() elements.ScoreReport {
 		ReturnedMeeples: map[uint8][]uint8{},
 	}
 }
+
+func GetTestCustomPlacedTile(tileTemplate tiles.Tile, playerID uint8) elements.PlacedTile {
+	var placedFeatures []elements.PlacedFeature
+
+	// covnert features to placedFeature
+	for _, feature := range tileTemplate.Features {
+		placedFeatures = append(placedFeatures, elements.PlacedFeature{
+			Feature: feature,
+			Meeple: elements.Meeple{
+				MeepleType: elements.NoneMeeple,
+				PlayerID:   elements.ID(playerID)},
+		})
+	}
+
+	return elements.PlacedTile{
+		TileWithMeeple: elements.TileWithMeeple{
+			Features:  placedFeatures,
+			HasShield: false,
+		},
+		Position: elements.NewPosition(0, 0),
+	}
```

Comment:
I think HasShield is no longer required in elements.PlacedTile, because shield is now a feature modifier.
@bstrzelecki did we miss this in #38 ?

Reply:
Yup, this field is redundant as of #38.

Reply:
Am i supposed to do something with it, or what? It's not this pull request for.

Reply:
I think if it doesn't cause any problems, we can leave it for now